tests/k8s-azure/manifest/linux.json

```diff
@@ -7,7 +7,7 @@
             "orchestratorRelease": "1.14",
             "kubernetesConfig": {
                 "useCloudControllerManager": true,
-                "customCcmImage": "gcrio.azureedge.net/google_containers/cloud-controller-manager-amd64:v1.14.0-alpha.1",
+                "customCcmImage": "gcrio.azureedge.net/google_containers/cloud-controller-manager-amd64:v1.14.0-rc.1",
```

Comment:
I think you forgot to add `customHyperkubeImage: "gcrio.azureedge.net/google_containers/hyperkube-amd64:v1.14.0-rc.1"`

Reply:
oops, added now.